packages/components/src/custom-select-control/types.ts

```diff
@@ -24,15 +24,15 @@ type Option = {
 /**
  * The object returned from the onChange event.
  */
-type ChangeObject = {
+export type CustomSelectChangeObject< T extends CustomSelectOption > = {
```

Comment:
Do we need to export this one? Seems like we don't.

Reply:
It would be useful if user want to get the `ChangeObject` type, otherwise we have to use the following snippet to get the type:

<img width="548" alt="image" src="https://github.com/user-attachments/assets/e616364c-9dfc-4efd-bfdb-135276b22b56">

Reply:
Right 👍 

Exporting it here doesn't export it from the package, though. And if we're not importing it inside the package, then it's effectively unnecessary to export it, IMHO. We can always start exporting it if we need to in the future.

Reply:
Yes, at the moment we are avoiding public type exports as much as possible, thanks 🙏

Reply:
ok, removed